aries-site/src/examples/templates/forms/RequiredFieldsExample.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Select,
+  Text,
+  TextInput,
+  TextArea,
+} from 'grommet';
+import { CircleAlert } from 'grommet-icons';
+
+const superPower = ['Flying', 'Sky Runner', 'Invisibility'];
+const weakness = ['Fire', 'PB & J', 'Kryptonite'];
+
+const RequiredFormField = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={required ? `${label}*` : label}
+      required={required}
+      {...rest}
+    />
+  );
+};
+
+RequiredFormField.propTypes = {
+  required: PropTypes.bool,
+  label: PropTypes.string,
+};
+
+const FormContainer = ({ ...rest }) => {
+  return (
+    <Box background="background-front" border round="small" overflow="hidden">
+      <Box flex pad={{ horizontal: 'medium', vertical: 'medium' }} {...rest} />
+    </Box>
+  );
+};
+
+const Error = ({ children, ...rest }) => {
+  return (
+    <Box align="center" background="background-front" gap="xsmall" direction="row"  {...rest} >
```

Comment:
Does this need "background-front"? feels like we don't need any background here

Reply:
some reason it was giving the validation red color

Reply:
Let's find out the reason. May be behavior we don't want.